source/conf.py

```diff
@@ -199,6 +199,31 @@ def setup(app):
     "administration/data-retention": "https://docs.mattermost.com/comply/data-retention-policy.html",
     "administration/custom-terms-of-service": "https://docs.mattermost.com/comply/custom-terms-of-service.html",
     "administration/mobile-changelog": "https://docs.mattermost.com/deploy/mobile-app-changelog.html",
+    "administration/config-settings.html#push-notification-contents": "https://docs.mattermost.com/configure/configuration-settings.html#push-notification-contents",
```

Comment:
Non-blocking, but should the redirect URL be on a new line?

Reply:
Good question! We can split the redirects up for readability, but it's not required.